Lib/copy.py

```diff
@@ -258,7 +258,7 @@ def _keep_alive(x, memo):
 
 def _reconstruct(x, memo, func, args,
                  state=None, listiter=None, dictiter=None,
-                 deepcopy=deepcopy):
+                 *, deepcopy=deepcopy):
```

Comment:
Would you explain how forcing `deepcopy` argument to be key-word argument would fix the bug?

Reply:
`__reduce__` is handled by doing `_reconstruct(obj, memo, *obj.__reduce__())`, so the bug was that a 6th parameter got assigned to `deepcopy`. Instead you get a `TypeError`.